func.lua

```diff
@@ -7,15 +7,21 @@ OPTIONS = {
   high = 6,
   length = 2,
   release = .2,
-  notes = {[0]=true, false, false, false, true, false, false, false, true, false, false, false},
+  tags = {[0]=false, false, false, false, false, false, false, false},
+  notes = {[0]=true, false, false, false, true, false, false, true, false, false, false, false},
```

Comment:
> Fixed bug where G# was enabled by default instead of G

This was not actually a bug. :slightly_smiling_face: 

I chose C-E-G# instead of the more harmonically natural C-E-G because in C-E-G# each note is equally spaced (4 semitones apart). This way, no sample will have to be corrected more than 2 semitones.

```suggestion
  notes = {[0]=true, false, false, false, true, false, false, false, true, false, false, false},
```